gating/generate_release_notes/release_notes_dockerfile

```diff
@@ -8,9 +8,8 @@ RUN echo "jenkins ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 
 RUN apt-get install -y pandoc
 
-# use released versions of the differs when they are available.
 RUN pip install rpc_differ==0.3.0 reno==2.5.1
-RUN pip install git+https://github.com/major/osa_differ@0.3.1
+RUN pip install git+https://github.com/major/osa_differ@0.3.2
```

Comment:
Could we use pypi for this rather than GitHub? https://pypi.python.org/pypi/osa_differ

Reply:
Maybe but that's not something I think should be addressed here. This patch is working code that is a back port of something already merged in both master and newton.

Is there a material downside with leaving it how it is? @hughsaunders may be able to explain why it was done that way. If this needs changing to install from PyPi it should be done as a separate commit.